ui/tests/integration/components/auth/page-test.js

```diff
@@ -112,6 +123,7 @@ module('Integration | Component | auth | page', function (hooks) {
 
       await this.renderComponent();
       await fillIn(AUTH_FORM.method, authType);
+      // await fillIn(AUTH_FORM.selectMethod, authType);
```

Comment:
don't mind these...i just got tired of undoing/redoing depending on which form component was being tested..